source/index.html

```diff
@@ -1,12 +1,208 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Кэт энерджи</title>
-  </head>
-  <body>
+<html class="page" lang="ru">
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page__body">
+
+  <header class="main-header">
+    <a class="main-header__logo">
+      <img src="" width="34" height="38" alt="Кэт Энерджи">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
```

Comment:
Я для лучшей читаемости рекомендовал бы писать вот так:
```suggestion
      <button class="main-nav__toggle" type="button">
        <span class="visually-hidden">Открыть меню</span>
      </button>
```

Все-таки находить тег внутри другого тега на одной строке может быть немного неудобно

Reply:
согласна, исправила